src/client/pages/AssessmentSection/DataTable/hooks/useGetTableData.ts

```diff
@@ -50,16 +51,29 @@ const useTableNames = (props: Props): Array<string> => {
 
 export const useGetTableData = (props: Props) => {
   const { assessmentName, countryIso, cycleName, table } = props
+  const country = useCountry(countryIso)
   const { odp } = table.props
 
   const dispatch = useAppDispatch()
   const tableNames = useTableNames(props)
 
+  // if we are using original data point, and we are not in table 1a extent of forest, we need to merge odp data for calculations
+  const mergeOdp =
```

Comment:
what if we are in table 1b and we're fetching table 1b? should mergeOdp be false ?

Reply:
mergeOdp is only false if we are hiding ODPs

Reply:
https://user-images.githubusercontent.com/5508251/218316297-458662ea-11a2-478d-9823-0ca4bdfeca86.mov

we cannot hide odps in 1b like we can in 1a

Reply:
what if country.props?.forestCharacteristics.useOriginalDataPoint is false and table.props.name !== TableNames.extentOfForest and tableNames.includes(TableNames.extentOfForest), then data fetched will include extent of forest with mergeOdp to false ?

Reply:
Table 1b:
Case1:
Disable ODPs:
```
mergeOdp false
country.props?.forestCharacteristics.useOriginalDataPoint false
tableNames.includes(TableNames.extentOfForest) true
```

Case2:
Enable ODPs:
```
mergeOdp true
country.props?.forestCharacteristics.useOriginalDataPoint true
tableNames.includes(TableNames.extentOfForest) true
```

This is correct, no?

If country disabled ODPs, the validations and calculations should NOT use ODP data?

Reply:
country.props?.forestCharacteristics.useOriginalDataPoint only applies to forestCharacteristics. all calculations involving 1a must include odp

Reply:
So, the correct condition is, we always mergeOdp unless we are in 1b and odps disabled?

`table.props.name !== TableNames.extentOfForest && country.props?.forestCharacteristics.useOriginalDataPoint`